Interop/Runtime/src/main/kotlin/kotlinx/cinterop/StableRef.kt

```diff
@@ -16,7 +16,7 @@
 
 package kotlinx.cinterop
 
-@Deprecated("Use StableRef<T> instead", ReplaceWith("StableRef<T>"))
+@Deprecated("Use StableRef<T> instead", ReplaceWith("StableRef<T>"), DeprecationLevel.ERROR)
 typealias StableObjPtr = StableRef<*>
```

Comment:
I'd rename StableRef to StableReference if there's a possibility

Reply:
How about `StrongReference`? (Or `GlobalReference` as the JNI named it). I'm not sure what "Stable" in `StableRef` means.

Reply:
Renaming is possible, but this is definitely out-of-scope for this PR.
Also it is not clear whether renaming makes sense here without redesigning.